### PR TITLE
[16.0][MIG] mrp_flattened_bom_xlsx_subcontracting_cost: Migration to 16.0

### PR DESCRIPTION
**Pull request by PauBForgeFlow**

### CODE REVIEW

**Review by BernatPForgeFlow — APPROVED** [review]
Functional and code review OK!

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+odoo-addon-mrp_flattened_bom_xlsx_direct_materials_cost@git+https://github.com/OCA/manufacture-reporting.git@refs/pull/116/head#subdirectory=setup/mrp_flattened_bom_xlsx_direct_materials_cost
```
Comment (LoisRForgeFlow) [inline test-requirements.txt]: @PauBForgeFlow Could you remove the test-requirements and rebase your PR? All dependencies are now merged

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**